index.html

```diff
@@ -40,6 +42,188 @@ <h4>I’am a Software Developer</h4>
+                </div>
+            </div>
+        </div>
+        <div class="card-work">
+            <div class="post-right-block">
+                <h1 class="card-post-title">Profesional Art Printing Data</h1>
+                <p class="supporting-text">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been
+                    the industry's standard</p>
+                    <ul class="post-tags-2">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+            </div>
+            <div class="btn-action">
+                <div class="post-btn-enable-2">
+                    <h1 class="btn-text">See Project</h1>
+                </div>
+            </div>
+        </div>
+        <div class="card-work">
+            <div class="post-right-block">
+                <h1 class="card-post-title">Profesional Art Printing Data</h1>
+                <p class="supporting-text">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been
+                    the industry's standard</p>
+                    <ul class="post-tags-2">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+            </div>
+            <div class="btn-action">
+                <div class="post-btn-enable-2">
+                    <h1 class="btn-text">See Project</h1>
+                </div>
+            </div>
+        </div>
+        <div class="card-work">
+            <div class="post-right-block">
+                <h1 class="card-post-title">Profesional Art Printing Data</h1>
+                <p class="supporting-text">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been
+                    the industry's standard</p>
+                    <ul class="post-tags-2">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+            </div>
+            <div class="btn-action">
+                <div class="post-btn-enable-2">
+                    <h1 class="btn-text">See Project</h1>
+                </div>
+            </div>
+        </div>
+    </div>
+
+</section>
+<!-- Recent Work End -->
+
+<!-- About Section Start -->
+<section class="rectangle">
+    <div class="about">
+        <h1 class="about-title">About me</h1>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what you see and have a project you
+            need coded, don’t hestiate to contact me.</p>
+        <div class="about-btn"><span class="about-btn-title">Get My Resume</span></div>
```

Comment:
- [ ] Nice Working on your `About Section`, kindly make the `Get My Resume`  a link `a` or `button` tag, for better use of HTML semantics.